juju/paths/paths_notwindows.go

```diff
@@ -0,0 +1,8 @@
+// Copyright 2016 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+// +build !windows
+
+package paths
+
+var Defaults = Nix
```

Comment:
`Defaults` seems dangerous to me. It could easily end up being used inappropriately.

Would you mind seeing how hard it is to do without the existence of Defaults. I'd be a lot happier if we had to think about which variant to use rather than automatically (and sometimes incorrectly) using Defaults.